### ISSUE TITLE
Versions seem wrong.

### ISSUE THREAD

**Issue by JoshCheek** [garybernhardt/selecta#68]
Selecta says its version is 0.0.2

```sh
> ag VERSION.\*=
> ./selecta  --version
```

Homebrew says it is 0.0.3

```sh
> brew info selecta | ag \\d\\.\\d\\.\\d
```

There is a tag for 0.0.3 with 42 commits on HEAD since then

```sh
> git tag
> git log v0.0.3^..HEAD --format='%C(yellow)%h%Cred%d%Creset %s '
```

-----

![screenshot 2015-01-05 10 21 09](https://cloud.githubusercontent.com/assets/77495/5616732/ad14195e-94c4-11e4-9e30-df687b3c382e.png)

**Comment by garybernhardt** [comment]
Yep, that's wrong. I've screwed this up with other projects in the past as well. I always forget to increment the number in two places. Do you know of any form of automation that makes this problem go away?

**Comment by JoshCheek** [comment]
I don't, I've never put anything on homebrew.

**Comment by garybernhardt** [comment]
Damn. I've made this mistake in non-homebrew projects too. It's just the mismatch between a VERSION constant in the code and the name of the tag. Maybe I'll make a little release script in bin/ that seds the code, commits, and creates a tag.

**Comment by garybernhardt** [comment]
I created that script and tagged 0.0.4 with it. https://github.com/garybernhardt/selecta/commit/b2b8d87d83042e733fcdddf9efeecd9abe4b5c50 I'm going to fix another bug and then push 0.0.5 to Homebrew.
